### ISSUE TITLE
RC10 release: DatePicker with "de" - console error

### ISSUE THREAD

**Issue by vladitasev** [SAP/ui5-webcomponents#2497]
![image](https://user-images.githubusercontent.com/15844574/99641499-076a6f00-2a53-11eb-89a3-bc22cd617e04.png)

when using DatePicker with "de" locale.

**Comment by ericwdq** [comment]
@vladitasev  Also tested 'zh_CN', throw the some error.

**Comment by vladitasev** [comment]
Hi Eric,

I debugged the problem by following the steps:
```
yarn create react-app my-app --template @ui5/webcomponents-react
```
and
```
 import {
  setLanguage
} from “@ui5/webcomponents-base/dist/config/Language.js”;

useEffect(() => {
    setLanguage(“de”);
  }, []);
```

The problem is that the `setLanguage` function is `async` and the code above does not await for it to finish. What `setLanguage` does internally is to fetch translation .json files and the CLDR .json file for the new language. When you don't await for it in `useEffect`, it immediately executes the App and returns the error. 

There are a couple of solutions:
 - Start with a configured language directly ('de' instead of the default 'en')
 - Call `setLanguage` before the whole lifecycle, f.e. if you do this:

![image](https://user-images.githubusercontent.com/15844574/99954653-cabcb180-2d8b-11eb-92b9-ff0c5387ae66.png)

it will work. I do realize this is a hack and not the intended way to go, just adding it here for demonstration purposes. When you call it like this, it will be executed on module import and will probably have finished before App is rendered. Of course this is not to be expected to work productively.

- Find out how to run async useEffect hooks in React. I'm not particularly into the topic so I cannot tell the best way to integrate async functions into the react lifecycle. I tried making the useEffect call async, and it didn't work. It only accepts a sync function as an argument.

On a general note, the `setLanguage` API is mainly intended to be used when reacting to the user's input. For example, having "Change language" menu or "flags" in your app and when the user clicks on a new language, you call setLanguage (and await for the async work to be done). It's not intended to set the language first thing when booting the app without user interaction - using the Configuration to start with the right language is usually the way to go.

Regards,
Vladi
